src/games/strategy/engine/framework/map/download/DownloadCoordinator.java

```diff
@@ -42,18 +42,18 @@ private final Thread createDownloadFileStarterThread() {
           ThreadUtil.sleep(50);
 
           if (shouldShowDownloadsFinishedPrompt()) {
-
             // TODO: we ignore the threading problem here, we may enter this code block multiple times.
             // It's a difficult situation to create for the user, and the worst case is multiple map download
             // complete dialogs appearing - which is not a big deal.
             downloadPromptAlreadyShown = true;
-            Runnable confirmAction = () -> {
-              if (downloadCompleteAction.isPresent()) {
-                downloadCompleteAction.get().run();
-              }
-            };
             SwingComponents.promptUser("Map Downloads Completed",
-                "Map downloads are complete, would you like to continue downloading more maps?", confirmAction);
+                "Map downloads are complete, would you like to continue downloading more maps?", () -> {}, // do nothing, keep the window open
+                () -> { // user clicked no, close the window
+                  if (downloadCompleteAction.isPresent()) {
+                    downloadCompleteAction.get().run();
+                  }
+                }
+            );
```

Comment:
The lib function is meant to be easy enough to follow. I'd rather fix the API, or let this one be a 'know your libs'. I do not want to have to label the params every time we use this function.